sldtk/helpers.py

```diff
@@ -89,7 +89,7 @@ def parse_input(config):
                     help="Number of slices to average to create the intensity "
                          "profile.")
     ap.add_argument("-t", "--threshold",
-                    type=_uint8,
+                    type=int,
```

Comment:
Type too broad. e.g. "-1337" would be a valid threshold which, assuming by 16-bit we're talking about uint16, is not desirable. If support for uint8 is to be kept (see below), I propose threshold and bias arguments are converted to fractions to remain agnostic of a given image's dtype.